src/com/Jacksonnn/DCCore/Configuration/DiscordWebhook.java

```diff
@@ -0,0 +1,369 @@
+
```

Comment:
We can ignore all of this and use DiscordSRV's API, which we already do. Refer to the /suggest command